InventoryBuilder.py

```diff
@@ -409,5 +420,28 @@ def get_nsxt_mgmt_cluster(self) -> dict:
                         'target': nsxt_mgmt_plane.target,
                         'token': nsxt_mgmt_plane.token,
                     }
-        self.iterated_inventory[str(self.iteration)]['nsxt_resources'] = tree
+        self.iterated_inventory[str(self.iteration)]['nsxt_mgmt_cluster'] = tree
+        return tree
+
+    def expose_nsxt_mgmt_nodes(self) -> dict:
+        tree = dict()
+        for nsxt_entry in self.nsxt_dict:
+            nsxt_mgmt_plane = self.nsxt_dict[nsxt_entry]
+            if not nsxt_mgmt_plane:
+                continue
+            tree[nsxt_mgmt_plane.target] = dict()
+            for nsxt_adapter in nsxt_mgmt_plane.adapter:
```

Comment:
```suggestion
            for nsxt_adapter_object in nsxt_mgmt_plane.adapter:
```
Either you add `_object` here too, or you remove it in L263

Reply:
outdated